### ISSUE TITLE
Mapping rules related to incorrect usage of landmarks as WCAG 1.3.1 and WCAG 2.4.1 failures

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#850]
[ARIA11: Using ARIA landmarks to identify regions of a page](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/ARIA11) is one of the sufficient techniques to meet the success criteria WCAG 1.3.1 and WCAG 2.4.1. 

WCAG specifies that it is not mandatory to use this technique 

If an author chooses to use this technique and fails to implement them in the right way, we believe the following rules (_which only check for the correctness of the implementation of landmarks_) can be treated as WCAG 1.3.1 and WCAG 2.4.1 failures 

1. landmark-main-is-top-level
2. landmark-banner-is-top-level
3. landmark-contentinfo-is-top-level
4. landmark-complementary-is-top-level
5. landmark-no-duplicate-banner
6. landmark-no-duplicate-contentinfo

**Comment by WilcoFiers** [comment]
This would be inconsistent with Deque's interpretation of WCAG 2.0. Failing to implement a technique is **not** a violation of WCAG 2.0.

> Techniques are informative—that means they are not required. The basis for determining conformance to WCAG 2.0 is the success criteria from the WCAG 2.0 standard—not the techniques.
> Source: https://www.w3.org/TR/UNDERSTANDING-WCAG20/understanding-techniques.html#ut-understanding-techniques-informative-head

There is nothing in WCAG 2.0 that suggests landmarks have to be implemented correctly if they are used. You can just as well have a poorly implemented landmark structure, but have your headings be marked up appropriately. That would pass both 2.4.1 and 1.3.1. So while I agree that failing those rules **may be** violation of WCAG 2.0, they aren't in 100% of the cases, and so there's a likely chance of false positives coming from such a change.
